Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -479,71 +478,70 @@ NumberCombinedWithDurationUnit: !nestedRegex
 AllRegex: !simpleRegex
-  def: \b(?<all>(all|full|whole)(\s+|-)(?<unit>jaar|maand|week|dag))\b
+  def: \b(?<all>((ge)?hele|volledige|ganse)(\s+|-)(?<unit>jaar|maand|week|dag))\b
 HalfRegex: !simpleRegex
-  def: (((een)\s*)|\b)(?<half>half\s+(?<unit>jaar|maand|week|dag|uur))\b
+  def: (((een)\s*)|\b)(?<half>(half|halve)\s+(?<unit>jaar|maand|week|dag|uur))\b
 ConjunctionRegex: !simpleRegex
-  def: \b((and(\s+for)?)|with)\b
+  def: \b((en(\s+voor)?)|plus)\b
 HolidayRegex1: !nestedRegex
-  def: \b(?<holiday>pasen|kerst|kerstavond|xmas|thanksgiving|halloween|yuandan|oudejaarsavond|silvester|silvesteravond)(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?\b
+  def: \b(?<holiday>goede\s+vrijdag|pasen|kerst|kerstavond|kerstmis|thanksgiving|halloween|oudejaarsavond|nieuwjaar|nieuwjaarsdag|bevrijdingsdag)(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?\b
   references: [ YearRegex, RelativeRegex ]
 HolidayRegex2: !nestedRegex
   def: \b(?<holiday>nationale dodenherdenking|nationale herdenking|dodenherdenking|dag van de leraar|dag van de arbeid)(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?\b
   references: [ YearRegex, RelativeRegex ]
 HolidayRegex3: !nestedRegex # -dag suffix
-  def: (?<holiday>(valentijns|valetijns|nieuwjaars|prinsjes|konings|koninginne|bevrijdings|hemelvaarts|eerste kerst|1e kerst|tweede kerst|2e kerst|vaders|vader|moeders|moeder|meisjes|amerikaanse onafhankelijkheids|onafhankelijkheids|dankzeggings|bosmarmotten|inauguratie|boomplant|boomfeest|vrijgezellen|nederlandse veteranen|veteranen)\s*(dag))(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?
+  def: (?<holiday>(valentijn|valentijnsdag|nieuwjaarsdag|prinsjesdag|koningsdag|koninginnedag|bevrijdingsdag|hemelvaartsdag|eerste kerstdag|1e kerst|tweede kerstdag|2e kerstdag|vaderdag|moederdag|meisjesdag|amerikaanse onafhankelijkheidsdag|onafhankelijkheidsdag|nederlandse veteranendag|veteranendag))(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?
   references: [ YearRegex, RelativeRegex ]
 AMTimeRegex: !simpleRegex
-  def: (?<am>'s morgens)
+  def: (?<am>('s morgens|'s ochtends)|in\s+de\s+(morgen|ochtend))
 PMTimeRegex: !simpleRegex
-  def: \b(?<pm>'s middags|'s avonds|'s nachts)\b
+  def: \b(?<pm>('s middags|'s avonds|'s nachts)|in\s+de\s+(middag|avond|nacht))\b
 InclusiveModPrepositions: !simpleRegex
-  def: (?<include>((on|in|at)\s+or\s+)|(\s+or\s+(on|in|at)))
+  def: (?<include>((in|tegen|tijdens|op)\s+of\s+)|(\s+of\s+(in|tegen|tijdens|op)))
 BeforeRegex: !nestedRegex
-  def: (\b{InclusiveModPrepositions}?(before|in\s+advance\s+of|prior\s+to|(no\s+later|earlier|sooner)\s+than|ending\s+(with|on)|by|till|til|until|(?<include>as\s+late\s+as)){InclusiveModPrepositions}?\b\s*)|(?<!\w|>)((?<include><=)|<)
+  def: (\b{InclusiveModPrepositions}?(voor|vóór |vooraf(gaan)?\s+aan|(niet\s+later|vroeger|eerder)\s+dan|eindigend\s+op\s+|tegen|tot|totdat|(?<include>zo\s+laat\s+als)){InclusiveModPrepositions}?\b\s*)|(?<!\w|>)((?<include><=)|<)
```

Comment:
Extra space after `vóór` necessary? (since it is not used with `voor`).

Reply:
it is not necessary, and it is removed in the update.

Reply:
Resolved.